formatter/formatSpecification.go

```diff
@@ -29,27 +29,47 @@ type formatter struct {
 	buffer bytes.Buffer
 }
 
+var nextItem gauge.Item
+
+func (formatter *formatter) SetNext(item gauge.Item) {
+	nextItem = item
+}
+
+func (formatter *formatter) GetNext() (item gauge.Item) {
+	return nextItem
+}
+
```

Comment:
`GetNext` is not idiomatic go code. Ref : https://golang.org/doc/effective_go.html#Getters

In any case, if you have `nextItem` in the struct, you need not have a `Getter`.